main.cpp

```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: knife
+//  action 1: Stab a Player
+knife.stab();
+//  action 2: Slash a Player
+knife.slash();
+//  action 3: Hold knife Menacingly
+knife.menace();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: Make a meow sound
+cat.meow();
+//  action 2: scratch an attacker
+cat.scratchAttacker();
+//  action 3: Jump
+cat.jump();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: box
+//  action 1: Move the box
+box.move();

```

Comment:
Who is moving the box? 
You need to pick actions that the Noun performs.  This action is performed by something other than the Noun ONTO the noun.
- [ ] revise